projects/containernetworking/plugins/Makefile

```diff
@@ -18,6 +27,8 @@ build: tarballs
 
 .PHONY: release
 release: tarballs
+	$(BASE_DIRECTORY)/release/copy_artifacts.sh $(COMPONENT) $(RELEASE_BRANCH) $(RELEASE)
```

Comment:
Same here, anything we pass in to the scripts, we should not export above. Exporting means that our child bash scripts would have access to the environment variable too I believe.

Reply:
The export of RELEASE_BRANCH was from a previous PR and I agree it isn't too cool. My survey of our shell files seems to indicate it will work without exporting that value. It is passed in all the scripts I could find that mentioned the variable.